### ISSUE TITLE
Change base project directory

### ISSUE THREAD

**Issue by Yimura** [YimMenu/YimMenuV2#6]

**Comment by Yimura** [comment]
Yeah sounds about right, unsure if we should keep it in the user's documents folder or use appdata again.

**Comment by maybegreat48** [comment]
I'm sticking with `AppData`, though we could use documents if we don't need backwards compatibility
